### PR TITLE
JBIDE-25208 LogsTest ITests stabilization

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by jkopriva** [comment]
green build on rhel: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/openshift.itests/440/jdk=openjdk-1.8,label=rhel7/

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/LogsTest.java [hunk 1]
```diff
@@ -131,6 +133,27 @@ public void shouldShowLogFromBuildPodContextMenu() {
...
+							new DefaultShell(OpenShiftLabel.Shell.LOGS_UNAVAILABLE);
+							new OkButton().click();
+							return false;
+						} catch (RedDeerException e) {
+							// catched intentionnally
+							System.err.println(e);
+						}
+						return true;
+					}}
+				, TimePeriod.VERY_LONG);
```
Comment (odockal) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/LogsTest.java]: What are you waiting for that long? 300 s? Is it necessary?

Comment (odockal) [reply itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/LogsTest.java]: Ah, I see. Good then.

**Comment by odockal** [comment]
+1.

**Pull request merged**